controllers/managedmcg_controller.go

```diff
@@ -54,6 +57,7 @@ const (
 	deployerCSVPrefix = "mcg-osd-deployer"
 	noobaaFinalizer   = "noobaa.io/graceful_finalizer"
 	noobaaName        = "noobaa"
+	OperatorNamespace = "openshift-data-federation"
```

Comment:
try to get namespace from the request

Reply:
Hard coding it to `redhat-data-federation` as of now.

Reply:
@v-harihar why we need to hardcode the namespace, can we get it from `ManagedMCGReconciler.namespace`

Reply:
+1